### PR TITLE
Add bob.blitz recipe

### PR DESCRIPTION
**Pull request by 183amir**

### CODE REVIEW

**Comment by 183amir** [comment]
Looks like I created a pull for this too early.

**Comment by jakirkham** [comment]
Yes, please wait for a bit. We need to fix an issue before we can add `bob.extension`.

**Comment by jakirkham** [comment]
So, the Linux version of `bob.extension` is released and I have restarted Circle CI accordingly. The Mac release is still pending so we may need to wait a bit longer for that.

File: recipes/bob.blitz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+source:
+  fn: bob.blitz-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/b/bob.blitz/bob.blitz-{{ version }}.zip
+  md5: 1d5db49ce0a5f64b8bd729e0455f52ab
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python -B setup.py install --single-version-externally-managed --record
+    record.txt
```
Comment (jakirkham) [inline recipes/bob.blitz/meta.yaml]: Looks like this slipped on to the next line. Should be on the previous line.

File: recipes/bob.blitz/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+requirements:
+  build:
+  - python
+  - setuptools
+  - bob.extension
+  - numpy
```
Comment (jakirkham) [inline recipes/bob.blitz/meta.yaml]: Are we linking against NumPy here?

**Comment by jakirkham** [comment]
The Mac release of `bob.extension` is out so Travis CI has been restarted.

**Comment by jakirkham** [comment]
The builds appear to be working, but there are a couple of things we still need to address.

File: recipes/bob.blitz/meta.yaml (hunk 1 above)
Comment (183amir) [inline recipes/bob.blitz/meta.yaml]: Ruamel.yaml library did that, I will fix it.

File: recipes/bob.blitz/meta.yaml (hunk 2 above)
Comment (183amir) [inline recipes/bob.blitz/meta.yaml]: Yes, should I do something extra for numpy?

Comment (jakirkham) [inline recipes/bob.blitz/meta.yaml]: Yes, add `x.x` after `numpy` in both `build` and `run`.

File: recipes/bob.blitz/meta.yaml (hunk 1 above)
Comment (pelson) [inline recipes/bob.blitz/meta.yaml]: Still needs doing.

Comment (183amir) [inline recipes/bob.blitz/meta.yaml]: Yes, I am a bit busy for the moment. I will get to it.

File: recipes/bob.blitz/meta.yaml (hunk 2 above)
Comment (183amir) [inline recipes/bob.blitz/meta.yaml]: Should I add `x.x` for every C library that we link against?

Comment (msarahan) [inline recipes/bob.blitz/meta.yaml]: Presently, that syntax applies only to numpy.  There is a lot of discussion on the topic - most recent at https://github.com/conda/conda-build/pull/848

**Comment by 183amir** [comment]
Ok. All checks are passed and comments are considered.

**Comment by jakirkham** [comment]
LGTM. @ocefpaf, any final thoughts on this one?

**Comment by ocefpaf** [comment]
Nope. Merge away! Thanks @183amir!!

**Comment by jakirkham** [comment]
Thanks @183amir! :smile:

**Pull request merged**